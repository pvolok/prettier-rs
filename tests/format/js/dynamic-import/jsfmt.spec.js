run_spec(import.meta, ["babel", "flow", "typescript"], {
  errors: {
    flow: ["assertions.js"],
    acorn: ["assertions.js"],
    espree: ["assertions.js"],
    meriyah: ["assertions.js"],
  },
});
