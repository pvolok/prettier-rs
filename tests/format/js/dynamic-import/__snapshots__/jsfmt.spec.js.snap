// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`assertions.js [acorn] format 1`] = `
"Unexpected token (1:20)
> 1 | import("./foo.json", { assert: { type: "json" } });
    |                    ^
  2 |"
`;

exports[`assertions.js [espree] format 1`] = `
"Unexpected token , (1:20)
> 1 | import("./foo.json", { assert: { type: "json" } });
    |                    ^
  2 |"
`;

exports[`assertions.js [flow] format 1`] = `
"Unexpected token \`,\`, expected the token \`)\` (1:20)
> 1 | import("./foo.json", { assert: { type: "json" } });
    |                    ^
  2 |"
`;

exports[`assertions.js [meriyah] format 1`] = `
"Expected ')' (1:20)
> 1 | import("./foo.json", { assert: { type: "json" } });
    |                    ^
  2 |"
`;

exports[`assertions.js format 1`] = `
====================================options=====================================
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
import("./foo.json", { assert: { type: "json" } });

=====================================output=====================================
import("./foo.json", { assert: { type: "json" } });

================================================================================
`;

exports[`test.js format 1`] = `
====================================options=====================================
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
import("module.js");
import("module.js").then((a) => a);

=====================================output=====================================
import("module.js");
import("module.js").then((a) => a);

================================================================================
`;
